src/cmd/merge.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright © 2024 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/NovoNordisk-OpenSource/decentralized-tech-radar/Merger"
+	"github.com/spf13/cobra"
+)
+
+// mergeCmd represents the merge command
+var mergeCmd = &cobra.Command{
+	Use:   "merge <FilePath1> <FilePath2> [FilePath3, ...]",
```

Comment:
usually the optional args are as individual items
```suggestion
	Use:   "merge <FilePath1> <FilePath2> [FilePath3] ...",
```